### ISSUE TITLE
Questions about the human3.6m annotation file

### ISSUE THREAD

**Issue by Ixiaohuihuihui** [microsoft/multiview-human-pose-estimation-pytorch#2]
Hi, 
First of all, thanks for open sourcing this great work. I have some questions: 
(1) This work retrain 2D pose estimation network using 2D ground truth heatmap of Human3.6 dataset, right? Because other works only use state-of-the-art 2D human pose estimator to generate 2D labels for Human3.6M without using the 2D ground truth.
(2) Another question is that how to generate the h36m_train.pkl？The file contains "joints_2d" and "joints_3d", how to combine the image file with corresponding joints 2d or 3d information? 
Thanks in advance!

**Comment by CHUNYUWANG** [comment]
(1) Yes, we train the whole network, including the fusion parameters, on the H36M dataset. 

(2) The 2D and 3D joint locations are provided by the dataset. We will release a toolbox on processing the H36M dataset soon.

**Comment by Ixiaohuihuihui** [comment]
Thank you! Looking forward to the releasing news!

**Comment by CHUNYUWANG** [comment]
Please check this repo for preparing images and annotations
https://github.com/CHUNYUWANG/H36M-Toolbox

**Comment by Ixiaohuihuihui** [comment]
Thank you very much!

**Comment by ahsan3803** [comment]
@CHUNYUWANG Hi, I've npz and h5 format for Human3.6m dataset. IS there anyway to convert these format into pkl format?

**Comment by Ixiaohuihuihui** [comment]
@ahsan3803 The convert progress is simple. Just read the array from h5 file and write the array into pkl.
For example:
```
import h5py
with h5py.File('/home/dailh/Study/3D_human_estimation/3d-pose-baseline/data/h36m/S1/MyPoses/3D_positions/Directions 1.h5', 'r') as h5f:
    poses = h5f['3D_positions'][:]
with open('h36m_train.pkl', 'wb') as f:
        pkl.dump(poses, f)
```
However, the key issue is that how you get the same data keys in your annotation files...

**Comment by ahsan3803** [comment]
@Ixiaohuihuihui Hi, thanks so much for your kind response. Yeah you are right it's key issue. We can't input NPZ file also due to 2D and 3D joints/pose information. 
Am unable to get Human3.6m original dataset, I tried many times but my account is not verified yet. Is there any other way to get this dataset?

**Comment by Ixiaohuihuihui** [comment]
@ahsan3803 
Send an email to Human3.6m team. And you need to supply any information which can prove your relationship with your advisor.

**Comment by ahsan3803** [comment]
@Ixiaohuihuihui  Hi, kindly check your email I've sent an email to you

**Comment by zhhaochen** [comment]
> @ahsan3803
> Send an email to Human3.6m team. And you need to supply any information which can prove your relationship with your advisor.

Hi, I send an email to Human3.6m team but  no reply. Can you share the pkl files? Thank you!
